diff --git a/node_modules/@keystone-next/keystone/session/dist/keystone.cjs.dev.js b/node_modules/@keystone-next/keystone/session/dist/keystone.cjs.dev.js
index 2e7e8a0..b7e7c3f 100644
--- a/node_modules/@keystone-next/keystone/session/dist/keystone.cjs.dev.js
+++ b/node_modules/@keystone-next/keystone/session/dist/keystone.cjs.dev.js
@@ -84,7 +84,8 @@ function statelessSessions({
   maxAge = MAX_AGE,
   path = '/',
   secure = process.env.NODE_ENV === 'production',
-  ironOptions = Iron__default['default'].defaults
+  ironOptions = Iron__default['default'].defaults,
+  sameSite = 'lax'
 }) {
   return () => {
     if (!secret) {
@@ -121,7 +122,7 @@ function statelessSessions({
           httpOnly: true,
           secure,
           path,
-          sameSite: 'lax'
+          sameSite
         }));
       },
 
@@ -138,7 +139,7 @@ function statelessSessions({
           httpOnly: true,
           secure,
           path,
-          sameSite: 'lax'
+          sameSite
         }));
         return sealedData;
       }
diff --git a/node_modules/@keystone-next/keystone/session/dist/keystone.cjs.prod.js b/node_modules/@keystone-next/keystone/session/dist/keystone.cjs.prod.js
index 2e7e8a0..b7e7c3f 100644
--- a/node_modules/@keystone-next/keystone/session/dist/keystone.cjs.prod.js
+++ b/node_modules/@keystone-next/keystone/session/dist/keystone.cjs.prod.js
@@ -84,7 +84,8 @@ function statelessSessions({
   maxAge = MAX_AGE,
   path = '/',
   secure =         "production" === 'production',
-  ironOptions = Iron__default['default'].defaults
+  ironOptions = Iron__default['default'].defaults,
+  sameSite = 'lax'
 }) {
   return () => {
     if (!secret) {
@@ -121,7 +122,7 @@ function statelessSessions({
           httpOnly: true,
           secure,
           path,
-          sameSite: 'lax'
+          sameSite
         }));
       },
 
@@ -138,7 +139,7 @@ function statelessSessions({
           httpOnly: true,
           secure,
           path,
-          sameSite: 'lax'
+          sameSite
         }));
         return sealedData;
       }
diff --git a/node_modules/@keystone-next/keystone/session/dist/keystone.esm.js b/node_modules/@keystone-next/keystone/session/dist/keystone.esm.js
index 2e7e8a0..b7e7c3f 100644
--- a/node_modules/@keystone-next/keystone/session/dist/keystone.esm.js
+++ b/node_modules/@keystone-next/keystone/session/dist/keystone.esm.js
@@ -76,7 +76,8 @@ function statelessSessions({
   maxAge = MAX_AGE,
   path = '/',
   secure = process.env.NODE_ENV === 'production',
-  ironOptions = Iron.defaults
+  ironOptions = Iron.defaults,
+  sameSite = 'lax'
 }) {
   return () => {
     if (!secret) {
@@ -113,7 +114,7 @@ function statelessSessions({
           httpOnly: true,
           secure,
           path,
-          sameSite: 'lax'
+          sameSite
         }));
       },
 
@@ -130,7 +131,7 @@ function statelessSessions({
           httpOnly: true,
           secure,
           path,
-          sameSite: 'lax'
+          sameSite
         }));
         return sealedData;
       }
